helper.php

```diff
@@ -220,7 +220,7 @@ public function ensureAbsoluteId($id) {
      * @param Doku_Renderer_xhtml $R
```

Comment:
Please update the type in PHPDocs.
Can you add also PHPDocs in the functions where your add new arguments to the function?

(and just a suggestion: some IDEs can generate the PHPDocs for you (and perform many other inspections as well). For example you could have a look at https://www.dokuwiki.org/devel:intellij_idea )